addon/components/confirm-extension.js

```diff
@@ -43,7 +43,7 @@ export default Ember.Component.extend({
     },
     confirm: function() {
       this.set('confirmMode', false);
-      this.sendAction('confirmAction');
+      this.sendAction('confirmAction', this.get('actionParam'));
```

Comment:
I think `actionParam` is misunderstandable, as the direct connection between `confirmAction` and `actionParam` is missing. Maybe `confirmActionParam` would be a better choice?